### DISCUSSION TITLE
property 'sourseld' doesn't exist

### DISCUSSION THREAD

**Discussion by JoaoPedro77** [lnreader/lnreader#588] [Q&A] [answered]
sorry, i don't know exactly where i should ask it, but i downloaded the app today and when i was going to read, this message appeared: property 'sourseld' doesn't exist 
it was in Portuguese br source light novels brasil
i really want to read some novels, but this keeps happening when i try...
what should I do?

![Screenshot_20230402_140313_LNReader](https://user-images.githubusercontent.com/69486969/229367966-b65eae64-2180-48e2-be5f-9ef2256e0ab5.jpg)

**Comment by rajarsheechatterjee** [comment, marked as answer]
The issue has been fixed in the [latest release](https://github.com/LNReader/lnreader/releases/tag/v1.1.15).
